src/Model/ResetPasswordToken.php

```diff
@@ -58,6 +58,14 @@ public function getToken(): string
         return $this->token;
     }
 
+    /**
+     * Allow the token object to be safely persisted in a session.
+     */
+    public function clearToken(): void
+    {
+        $this->token = 'void';
```

Comment:
How about we set it to `null`.

And then in `getToken()`, we add a check: if `$this->token === null`, we throw a big exception that says:

> The token property can't be accessed after the clearToken() method has been called.

Or something like that.

Reply:
done